unit-tests/src/test/java/org/eclipse/collections/impl/map/fixed/TripletonMapTest.java

```diff
@@ -360,9 +360,9 @@ public void testEqualsAndHashCode()
     @Test
     public void testClone()
     {
-        MutableMap<Integer, String> map = new TripletonMap<>(1, "One", 2, "Two", 3, "Three");
         try
         {
+            MutableMap<Integer, String> map = new TripletonMap<>(1, "One", 2, "Two", 3, "Three");
```

Comment:
This is similar. I see no point reducing scope into the beginning of a try block.

Reply:
Reverted